packages/grafana-ui/src/components/BarGauge/BarGauge.tsx

```diff
@@ -363,6 +378,12 @@ export function calculateBarAndValueDimensions(props: Props): BarAndValueDimensi
     }
 
     valueWidth = width;
+
+    if (valueMode === BarGaugeValueMode.Hidden) {
```

Comment:
This could be an if...else so we don't do the above calculations if valueMode is hidden